worldengine/world.py

```diff
@@ -34,6 +35,8 @@ def __init__(self, name, width, height, seed, num_plates, ocean_level,
         self.step = step
         self.temps = temps
         self.humids = humids
+        self.gamma_curve = gamma_curve
```

Comment:
we should serialize these values. I opened an issue for that (#174) as we can do that also after having merged this patch